lib/groonga/command/parser/command/groonga-command-convert-format.rb

```diff
@@ -76,6 +77,12 @@ def parse_options!(argv)
                              "Available only in command format",
                              "[#{@pretty_print}]") do |boolean|
               @pretty_print = boolean
+
+            option_parser.on("--elasticsearch-version=VERSION",
+                             "Specify the Elasticsearch version",
+                             "Available only in elasticsearch format",
+                             "[#{@elasticsearch_version}]") do |version|
+              @elasticsearch_version = version
```

Comment:
It seems that it is better to describe what Elasticsearch versions are supported